wsd/FileServer.cpp

```diff
@@ -259,8 +259,7 @@ bool FileServerRequestHandler::isAdminLoggedIn(const HTTPRequest& request,
     Poco::Net::HTTPCookie cookie("jwt", jwtToken);
     // bundlify appears to add an extra /dist -> dist/dist/admin
     cookie.setPath(COOLWSD::ServiceRoot + "/browser/dist/");
-    cookie.setSecure(COOLWSD::isSSLEnabled() ||
```

Comment:
I wonder if it's good. What happens if coolwsd server is behind a reverse proxy, we reach the reverse proxy by https, and SSL termination is enabled?

Reply:
Can you please test with a reverse proxy to collect data, what happens?. My expectation the cookies will be sent, otherwise if it is not a reverse proxy the cookie is blocked by the browser.